docker/workstation_computer/docker-compose.yml

```diff
@@ -20,4 +20,5 @@ services:
       # - ../../calibration:/root/git/calibration
       - ../../tests/frankapy_control_test_scripts:/root/git/tests/frankapy_control_test_scripts
```

Comment:
probably the directory written here is wrong and because of that we cannot find the `frankapy_control_test_scripts` inside the mounted  folder. The `tests` folder should be removed.